README.md

```diff
@@ -47,9 +46,10 @@ git clone --recursive https://github.com/clara-genomics/AtacWorks.git
 * Install pip dependencies
 
     ```
-    pip install -r requirements.txt
+    cd AtacWorks && pip install -r requirements.txt
     ```
-* Optional -- Install macs2
+* Optional -- Install macs2.
```

Comment:
Change to : Only required if you want to use macs2 subcommands to call peaks based on peak probabilities generated by AtacWorks.

Reply:
done.